Sources/PointFree/Routers/GitHubRoutes.swift

```diff
@@ -19,25 +19,25 @@ public enum GitHubRoute: DerivePartialIsos {
 
 let gitHubRouter = [
 
-  GitHubRoute.iso.authorize
+  PartialIso.authorize
```

Comment:
for some strange reason I could not omit this `PartialIso` ... it's the only one that needed it!

Reply:
You need one to hint the compiler! I think you could hint by giving `gitHubRouter` an explicit type, too.

Reply:
but somehow it works for `TwitterRoute`!